tests/go/test_files/vet_good.go

```diff
@@ -1,5 +1,5 @@
 package main
 
 func main() {
-	return 1
+
```

Comment:
Probably consider placing the sample hello world program of go in here
Better than a blank function :sweat_smile: 
```
package main
import "fmt"
func main() {
    fmt.Printf("%q","hello world")
}
```

Rest looks good to go

Reply:
That's an unnecessary request, don't you think :wink:.

Reply:
Just a suggestion :smiley: 
My two cents for making coala better :)

Reply:
It is a reasonable request, given that our previous example good go file wasnt very good.
It is possible that a future version of go disallows an empty function.